src/Kernel/IQSharpEngine.cs

```diff
@@ -178,6 +178,17 @@ private async Task StartAsync()
             this.Workspace = await serviceTasks.Workspace;
             var references = await serviceTasks.References;
 
+            // Start registering magic symbols; we do this in the engine rather
```

Comment:
Would it make sense to  move this a little earlier?
Not sure this code reqires all other services to be ready.

Reply:
Good point, I think I can move this above the other awaits, too. Let me go on and do that.